xamarin/Xamarin.Forms
# Fix 8222 - [UWP] Make text in Editor control top aligned

### Description of Change ###

The text in the Editor control became vertically centered after #6463 introducing VerticalTextAlignment, as described in #8222. This fix sets the VerticalContentAlignment to VerticalAlignment.Top in the EditorRenderer's internal FormsTextBoxStyle, making the text top aligned again. 

### Issues Resolved ### 

- fixes #8222

### API Changes ###
 
 None

### Platforms Affected ### 

- UWP

### Behavioral/Visual Changes ###


None

### Before/After Screenshots ### 

Before:
![8222-before](https://user-images.githubusercontent.com/3807144/67753195-a41b3280-fa34-11e9-8afa-7521c984c33b.png)

After:
![8222-after](https://user-images.githubusercontent.com/3807144/67753227-b1d0b800-fa34-11e9-9c2c-78d70c348212.png)


### Testing Procedure ###

In UWP, go to Test Case 8222. Check that the shown Editor control's text is top-aligned.

### PR Checklist ###
